src/views/Alteration.vue

```diff
@@ -1,111 +1,113 @@
-          </p>
-        </section>
-
-        <AlterationSummary
-          class="mt-10"
-          :validate="getAppValidate"
-          @haveChanges="onAlterationSummaryChanges()"
-        />
-
-        <DocumentsDelivery
-          class="mt-10"
-          sectionNumber="1."
-          :validate="getAppValidate"
-          @valid="setDocumentOptionalEmailValidity($event)"
-        />
-
-        <TransactionalFolioNumber
-          v-if="showTransactionalFolioNumber"
-          class="mt-10"
-          sectionNumber="2."
-          :validate="getAppValidate"
-        />
-
-        <CertifySection
-          class="mt-10"
-          :sectionNumber="showTransactionalFolioNumber ? '3.' : '2.'"
-          :validate="getAppValidate"
-          :disableEdit="!isRoleStaff"
-        />
-
-        <!-- STAFF ONLY: Court Order/Plan of Arrangement and Staff Payment -->
-        <template v-if="isRoleStaff">
-          <CourtOrderPoa
+  <ViewWrapper>
+    <section class="pb-10" id="alteration-view">
+      <!-- Company Information page-->
+      <v-slide-x-transition hide-on-leave>
+        <div v-if="!isSummaryMode">
+          <header>
+            <h1>Company Information</h1>
+          </header>
+
+          <section class="mt-6">
+            You are legally obligated to keep your company information up to date. Necessary fees will be
+            applied as updates are made.
+          </section>
+
+          <YourCompany class="mt-10" />
+
+          <CurrentDirectors class="mt-10" />
+
+          <ShareStructures class="mt-10" />
+
+          <Articles class="mt-10" />
+        </div>
+      </v-slide-x-transition>
+
+      <!-- Review and Certify page -->
+      <v-slide-x-reverse-transition hide-on-leave>
+        <div v-if="isSummaryMode && showFeeSummary">
+          <header>
+            <h1>Review and Certify</h1>
+          </header>
+
+          <section class="mt-6">
+            <p id="intro-text">
+              Review and certify the changes you are about to make to your company. Certain changes require
+              an Alteration Notice which will incur a {{filingFeesPrice}} fee. Choosing an alteration date
+              and time in the future will incur an additional {{futureEffectiveFeesPrice}} fee.
+            </p>
+          </section>
+
+          <AlterationSummary
             class="mt-10"
-            :sectionNumber="showTransactionalFolioNumber ? '4.' : '3.'"
-            :autoValidation="getAppValidate"
+            :validate="getAppValidate"
+            @haveChanges="onAlterationSummaryChanges()"
           />
 
-          <StaffPayment
+          <DocumentsDelivery
             class="mt-10"
-            :sectionNumber="showTransactionalFolioNumber ? '5.' : '4.'"
-            @haveChanges="onStaffPaymentChanges()"
+            sectionNumber="1."
+            :validate="getAppValidate"
+            @valid="setDocumentOptionalEmailValidity($event)"
           />
-        </template>
```

Comment:
This hasn't changed.  It's just Github being dumb with how it recognizes changes.